ieeg_filtered_speech/sub-cm8/ses-01/.ipynb_checkpoints/sub-cm8_ses-01_ieeg-checkpoint.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
what is ieeg-checkpoint?

Reply:
accidental addition - it's a jupyter auto-generated thing to retain state of your session...will remove in next push